js/packages/berty-storybook/chat/shared-components/Message.tsx

```diff
@@ -111,10 +112,22 @@ export const Message: React.FC<MessageProps> = ({
 				</View>
 				<View style={[payload.isMe && row.item.bottom]}>
 					{!state.length ? (
-						<Text style={[text.color.grey, _styles.dateMessage]}>
-							{formatTimestamp(new Date(payload.sentDate))}{' '}
-							{payload.isMe && (payload.acknowledged ? 'Sent' : 'Sending..')}
-						</Text>
+						<View style={[row.left, { alignItems: 'center' }]}>
+							<Text style={[text.color.grey, padding.right.scale(5), _styles.dateMessage]}>
+								{formatTimestamp(new Date(payload.sentDate))}{' '}
+							</Text>
+							{payload.isMe && (
+								<Icon
+									name={payload.acknowledged ? 'navigation-2' : 'navigation-2-outline'}
+									width={12}
+									height={12}
+									fill={color.blue}
+								/>
+							)}
+							<Text style={[text.color.grey, padding.left.scale(3), _styles.dateMessage]}>
```

Comment:
I think we should remove the text if we add the icon

Reply:
I think it's good if we have both for more clarity, if we have just an icon the user may have a different understanding from ours no ?

Reply:
my bad I had bad memory of the mockups